scripts/updateTemplate.mjs

```
@@ -0,0 +1,105 @@
+  }
+
+  Object.entries(defaultScripts).forEach(([name, value]) => {
+    pkgData.scripts[name] = value
+  })
+
+  pkgData.devDependencies['concurrently'] = '^8.2.0'
+
+  Object.keys(pkgData.dependencies)
+    .filter((name) => name.includes('@opentiny/tiny-engine'))
+    .forEach((name) => (pkgData.dependencies[name] = version))
+  Object.keys(pkgData.devDependencies)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)